package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "main": "dist/index.js",
   "module": "esm/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
most of the packages in gmod do not specify this types field explicitly, generally it seems like the types are automatically found without this line? could possibly remove